MigrateServicePreferenceFromLegacy/handler.ts

```diff
@@ -136,9 +133,18 @@ export const MigrateServicePreferenceFromLegacy = (
                   ),
             _ => te.taskEither.of(true)
           )
-      )
-    )
-    .chain(m => a.array.sequence(te.taskEither)(m))
+      );
+      return a.array
+        .sequence(te.taskEither)(tasks)
+        .map(_ => {
+          tracker.profile.traceMigratingServicePreferences(
+            migrateInput.oldProfile,
+            migrateInput.newProfile,
+            "DONE"
+          );
+          return _;
+        });
```

Comment:
Not sure about this implementation: in case of multiple `ServicePreferences` we're going to raise an event per each one of them (no matter if the task returned  `true` or `false`).
So maybe we can add the serviceId and the result ( _ ) to have a better understanding of the situation

Reply:
[These tests](https://github.com/pagopa/io-functions-app/pull/185/commits/9675560a950a8f60299f4fb9e322cd1d8290ef83) say it's ok

![](https://media0.giphy.com/media/MBCyiM4HIITb1E316S/giphy.gif)